projects/canopy/src/lib/accordion/accordion-panel-heading/accordion-panel-heading.component.html

```diff
@@ -8,7 +8,11 @@
     [attr.aria-controls]="_panelId"
     (click)="toggle()"
   >
+    <lg-icon *ngIf="!!iconName" [name]="iconName" class="lg-accordion__heading-icon"></lg-icon>
```

Comment:
I highly recommend against using `!!` as all it does is add overhead. The if statement by default converts the value to boolean.

Reply:
This doesn't apply anymore, as I used another approach to set a decorative icon.